docs/PacketFence_Installation_Guide.asciidoc

```diff
@@ -279,212 +216,237 @@ server, DHCP server, RADIUS server) using:
+ * Step 1 - Enforcement, choose 'RADIUS enforcement' - this will make PacketFence act as simple RADIUS server.
+ * Step 2 - Networks, make sure you define only one interface with the Management type. That network interface will be the one for which the Cisco 2960 access switch will talk to. The management interface of PacketFence and the Cisco 2960 should normally be in the same network.
+ * Step 3 - Database Configuration - provide the required information to properly create the PacketFence database
+ * Step 4 - PacketFence Configuration - provide your domain name, DHCP servers list and other required information
+ * Step 5 - Administration - provide the PacketFence's admin username and password to be used
+ * Step 6 - Fingerbank - provide your Fingerbank API key. Fingerbank is used to accurately identify Internet of Things (IoT) devices, medical devices, industrial and robotics equipment and more on your network. It is recommended to have a key for your PacketFence deployment.
+ * Step 7 - Configuration - start all services.
 
-[options="compact"]
-* Everyone behind an inline interface is on the same Layer 2 LAN
-* Every packet of authorized users goes through the PacketFence server increasing the server's load considerably: Plan ahead for capacity
-* Every packet of authorized users goes through the PacketFence server: it is a single point of failure for Internet access
-* Ipset can store up to 65536 entries, so it is not possible to have a inline network class upper than B
+Once all services are started, you will automatically be redirected to the PacketFence's web admin interface. It is located at https://@ip_of_packetfence:1443/admin
 
-This is why it is considered a poor man's way of doing access control. We have avoided it for a long time because of the above mentioned limitations. That said, being able to perform both inline and VLAN enforcement on the same server at the same time is a real advantage: it allows users to maintain maximum security while they deploy new and more capable network hardware providing a clean migration path to VLAN enforcement.
+Open that link and log in using the username/password specified in Step 5.
 
-Technical introduction to Out-of-band enforcement 
--------------------------------------------------
+Connecting PacketFence to Microsoft Active Directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Introduction
-~~~~~~~~~~~~
+Next we join the PacketFence server to your existing Microsoft Active Directory domain controller. From PacketFence's web admin interface, go in 'Configuration->Policies and Access Control->Domains->Active Directory Domain' and click on the 'Add domain' button. Provide the required fields. You will need an Active Directory administrative username and password to join the PacketFence server to your domain. Once all the information has been provided, click on the 'Save and join" button'.
```

Comment:
I'd add a tiny bit more details:

You will need an Active Directory administrative username and password (member of the domain admins) to join the PacketFence server to your domain.

Reply:
I'd also guide the user through the most important fields here so he gets them right (workgroup, domain DNS name)